app.css

```diff
@@ -0,0 +1,56 @@
+* {
+  direction: rtl;
+  padding:0px;
+  margin:0px;
+}
```

Comment:
2. another question - if ill write small code every time and wait for the evening to see your comment - its gonna be very slow? ill have to wait the whole night for you comment ..?